src/components/TabsDetails.jsx

```diff
@@ -0,0 +1,21 @@
+
+
+import React from 'react';
+
+function TabsDetails({tabName, handleTabEvents, currentTab}) {
+  return (
+         <div>
+             <button
+               onClick={(e) => handleTabEvents(e)}
```

Comment:
`onClick={handleTabEvents}` should also work.
Test this once.

Reply:
addressed